deploy/osd-ingress/routerreplicas-osd-20989/config.yaml

```
@@ -0,0 +1,6 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  matchExpressions:
```

We should also only do this on multi-az clusters, correct?
```suggestion
  matchExpressions:
      - key: api.openshift.com/multi-az
      operator: In
      values: "true"
```